test/types/range/hilde/count.good

```diff
@@ -3,42 +3,42 @@
 10..10
 1..10
 1..10
-1..0 by 10
+10..0 by 10
 10..19 by 10
 91..100 by 10 align 0
 10..100 by 10
 10..100 by 10
-1..0 by 10
+-50..-60 by 10
 -50..-41 by 10
 41..50 by 10 align 0
 -50..50 by 10
 -50..50 by 10
-1..0
+0..4294967295
 1073741825..2147483648
 0..1073741824
 1..2147483648
 0..2147483646
 4294967295..2147483648
-1..0 by 2
+0..4294967294 by 2
 1..2147483648 by 2 align 0
 0..2147483648 by 2
-1..0 by -1073741824
+0..3221225472 by -1073741824
 0..1073741823 by -1073741824 align 0
 1073741825..2147483648 by -1073741824
 0..2147483647 by -1073741824 align 0
 1..2147483648 by -1073741824
 0..2147483648 by -1073741824
 0..2147483648 by -1073741824
-1..0 by 1073741825
+0..3221225471 by 1073741825
 1073741824..2147483648 by 1073741825 align 0
 0..1073741824 by 1073741825
 0..2147483648 by 1073741825
 0..2147483648 by 1073741825
-1..0
-1..0 by 2
-1..0 by -1073741824
-1..0 by 1073741825
-1..0
-1..0 by 2
-1..0 by -1073741824
-1..0 by 1073741825
+-2147483648..2147483647
+-2147483648..2147483646 by 2
+-2147483648..1073741824 by -1073741824 align 1073741823
+-2147483648..1073741823 by 1073741825
+-9223372036854775808..9223372036854775807
+-9223372036854775808..9223372036854775806 by 2
+-9223372036854775808..9223372035781033984 by -1073741824 align 1073741823
+-9223372036854775808..9223372035781033983 by 1073741825
```

Comment:
I am suspicious about these added `align` clauses. However, if you looked at them and found them correct, that's fine by me.

Reply:
I've double-checked them and believe them to be correct.  That said, this last block of changes are likely to be non-portable due to the reliance on integer wraparound, so arguably should be removed.  I thought I'd check them in to see how non-portable they were before retiring them prematurely, though.

Reply:
Sounds good.